docs/pages/includes/database-access/db-configure-start-clickhouse.mdx

```diff
@@ -0,0 +1,62 @@
+{{ dbName="test" }}
+On the host where you will run the Teleport Database Service, start Teleport
+with the appropriate configuration.
+
+Note that a single Teleport process can run multiple different services, for
+example multiple Database Service agents as well as the SSH Service or Application
+Service. The step below will overwrite an existing configuration file, so if
+you're running multiple services add `--output=stdout` to print the config in
+your terminal, and manually adjust `/etc/teleport.yaml`.
+
+Generate a configuration file at `/etc/teleport.yaml` for the Database Service:
+
+```code
+$ teleport db configure create \
+   -o file \
+   --token=/tmp/token \
+   --proxy=<Var name="teleport.example.com" description="Your Teleport Proxy Service or Teleport Cloud tenant address"/> \
+   --name={{ dbName }} \
+   --protocol={{ dbProtocol }} \
+   --uri={{ databaseAddress }} \
+   --labels=env=dev
+```
+
+For ClickHouse integration with Native interface set the protocol value to `--protocol=clickhouse` installed of `--protocol=clickhouse-http` and point `--uri` to the
+ClickHouse Native interface listener.
```

Comment:
```suggestion
For ClickHouse integration with Native interface set the protocol value to `--protocol=clickhouse` instead of `--protocol=clickhouse-http` and point `--uri` to the
ClickHouse Native interface listener.
```
Or maybe give an actual example how to setup native

Reply:
thanks for suggestion.